utils/helpers.js

```diff
@@ -267,6 +269,19 @@ const unPauseAndPoll = async(ERC20Pausable) => {
 
     return false // paused
 }
+
+const pollForLastEvent = async(transactionReceipt) => {
+  for (let numberOfTries = 0; process.env.MAX_RETRY <= process.env.MAX_RETRY; numberOfTries++) {
+    const event = transactionReceipt.events.filter(
+      (e) => e.event === Constants.Events.ResponseCode
```

Comment:
You still have this hard coded, you should update the method signature to take in a 2nd param and use it

Reply:
Updated.